core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -247,33 +229,22 @@ angular.module('oppia').component('contributionsAndReview', {
 
       var _showTranslationSuggestionModal = function(
           suggestionIdToContribution, initialSuggestionId, reviewable) {
-        var _templateUrl = UrlInterpolationService.getDirectiveTemplateUrl(
-          '/pages/contributor-dashboard-page/modal-templates/' +
-          'translation-suggestion-review.directive.html');
         var details = ctrl.contributions[initialSuggestionId].details;
         var subheading = (
           details.topic_name + ' / ' + details.story_title +
           ' / ' + details.chapter_title);
-        $uibModal.open({
-          templateUrl: _templateUrl,
-          backdrop: 'static',
-          size: 'lg',
-          resolve: {
-            suggestionIdToContribution: function() {
-              return cloneDeep(suggestionIdToContribution);
-            },
-            initialSuggestionId: function() {
-              return initialSuggestionId;
-            },
-            reviewable: function() {
-              return reviewable;
-            },
-            subheading: function() {
-              return subheading;
-            }
-          },
-          controller: 'TranslationSuggestionReviewModalController'
-        }).result.then(function(resolvedSuggestionIds) {
+        let modalRef: NgbModalRef = NgbModal.open(
```

Comment:
If the variable is not being reassigned, this can be declared as `const`.

Reply:
Done